frontend/src/__tests__/integration/pages/projects/ProjectView.stories.tsx

```diff
@@ -27,6 +28,18 @@ export default {
         rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
           res(ctx.json(mockK8sResourceList([mockProjectK8sResource({})]))),
         ),
+        rest.delete(
```

Comment:
is there a reason you add the 403 error mock to the default story export, and not just the delete project story?

It doesn't really effect things, just a nit

Reply:
Because I thought it might be reused in other delete modal stories.